src/main/kotlin/com/safehill/kclient/util/Utils.kt

```diff
@@ -12,6 +12,7 @@ inline fun <T> runCatchingPreservingCancellationException(
     } catch (e: CancellationException) {
         throw e
     } catch (e: Exception) {
+        println("Error $e")
```

Comment:
What was the decision on logging in the framework?

Reply:
I will have that implemented as well in the next PR.